sheet.go

```diff
@@ -1183,6 +1215,35 @@ func (f *File) UnprotectSheet(sheet string) error {
 	return err
 }
 
+// ValidateSheetProtect provides a function to validate worksheet protect password.
+func (f *File) ValidateSheetProtect(sheet string, password string) (error, bool) {
```

Comment:
I propose to add the second optional parameter for the existing function `UnprotectSheet` (semantics of unlocking with password) instead of adding a new exported function `ValidateSheetProtect`. If the user-specified the second parameter is the wrong password, the error will be returned, if no second parameter has been specified, it will be forced to remove worksheet protection.

Reply:
I wil rewrite this.